core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -68,6 +68,26 @@
 
         <div role="form" class="form-horizontal">
           <div class="form-group">
+            <label class="col-lg-2 col-md-2 col-sm-2" translate="I18N_PREFERENCES_PREFERRED_DASHBOARD"></label>
+            <div class="col-lg-10 col-md-10 col-sm-10">
+              <div class="checkbox" style="padding-top: 0;">
+                <label>
+                  <input type="radio" ng-model="defaultDashboard" value="creator" ng-change="saveDefaultDashboard(defaultDashboard)">
```

Comment:
'creator' and 'learner' should be read, somehow, from constants.js, in order to tie the constants to a single source of truth. One possible way might be to do `ng-change="changeDefaultDashboardToCreator()"` and refer to those constants only in the JS.